management/server/http/policies_handler.go

```diff
@@ -261,22 +250,32 @@ func (h *Policies) GetPolicy(w http.ResponseWriter, r *http.Request) {
 func toPolicyResponse(account *server.Account, policy *server.Policy) *api.Policy {
 	cache := make(map[string]api.GroupMinimum)
 	ap := &api.Policy{
-		Id:          policy.ID,
 		Name:        policy.Name,
 		Description: policy.Description,
 		Enabled:     policy.Enabled,
 		Query:       policy.Query,
 	}
+	if policy.ID != "" {
+		id := policy.ID
+		ap.Id = &id
+	}
 	if len(policy.Rules) == 0 {
```

Comment:
policy.Rules should never be empty either, in this case we should panic or return nothing with a error 500

Reply:
Fixed